backend/sendmessage/src/set-scale.ts

```diff
@@ -1,35 +1,18 @@
-import { Config } from './types';
-import { getConnectionItem, getGroupItem } from './get-item';
-import { getResetVotesUpdateParams } from './reset-votes';
 import { broadcastState } from './broadcast-state';
+import { resetPersistedVotes } from './reset-votes';
```

Comment:
We have a name conflict here with `resetVotes`, that is also in the same file meaning "do a reset and announce it" and this operation which is just "do the database update for a reset". But I think I wanted to extract the database updates and forget about it, this could be one way to untangle this (though we would have two "resetVotes" in the code base then).